src/ofono/provider_ofono.hpp

```diff
@@ -179,6 +201,7 @@ class Bridge : public QObject, public PropertiesSource
     std::pair<QString, QString> network_name_;
     void (Bridge::*set_name_)();
 
+    QString modem_pattern_;
```

Comment:
I am not sure should be ```modem_pattern_``` stored directly as ```QRegularExpression``` or it will take more resources than ```QString```?